eng/pipelines/bicep-warnings.yml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '**/*.bicep'
+      - eng/pipelines/bicep-warnings.yml
+
+stages:
+  - stage: LintBicepFiles
+    displayName: Lint Bicep Files
+
+    pool:
+      vmImage: azsdk-pool-mms-ubuntu-2204-1espt
+
+    jobs:
+      - job: LintBicep
+        timeoutInMinutes: 20
+        steps:
+          - script: |
+              az bicep install
+              find . -name '*.bicep' -exec az bicep lint --file {} \; > lint-warnings.txt 2>&1
+            displayName: 'Install Bicep CLI and Lint Bicep Files'
+
+          - script: |
+              DATE=$(date +'%Y-%m-%d')
+              sed '/^$/N;/^\n$/D;' lint-warnings.txt > lint-warnings-${DATE}.txt
+            displayName: 'Rename Lint Warnings File with Date'
+
+          - task: PublishPipelineArtifact@1
+            inputs:
+              targetPath: 'lint-warnings-$(date +%Y-%m-%d).txt'
+              artifactName: 'bicep-lint-warnings-$(date +%Y-%m-%d)'
+
+          - task: AzureFileCopy@6
+            displayName: 'Upload Bicep Lint Warnings'
+            inputs:
+              sourcePath: 'lint-warnings-$(date +%Y-%m-%d).txt'
+              azureSubscription: 'Azure SDK Artifacts'
+              destination: AzureBlob
+              storage: azuresdkartifacts
+              containerName: 'bicep-lint-warnings'
+              blobPrefix: 'warnings/$(date +%Y-%m-%d)'
```

Comment:
```suggestion
              blobPrefix: 'bicep-lint-warnings/$(date +%Y-%m-%d)'
```

Reply:
Done in https://github.com/Azure/azure-sdk-for-js/pull/30904/commits/c62f166432ab5a59465eb85726c5661502e854c5